<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginUI</class>
 <widget class="QMainWindow" name="LoginUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>574</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LoginUI</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #131A2D;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>198</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>550</width>
           <height>420</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>550</width>
           <height>420</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #517078;
border-radius: 3px;
</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>40</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;
font: 18pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Đăng nhập</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="user_name">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>109</y>
            <width>351</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;
border-bottom: 2px solid white;
color: white;
</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="password">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>159</y>
            <width>351</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;
border-bottom: 2px solid white;
color: white;
</string>
          </property>
         </widget>
         <widget class="QPushButton" name="login_button">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>232</y>
            <width>351</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid white;
color: white;
font: 12pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Đăng nhập</string>
          </property>
         </widget>
         <widget class="QPushButton" name="register_button">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>290</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;
font: 10pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Đăng kí</string>
          </property>
         </widget>
         <widget class="QPushButton" name="fogot_button">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>290</y>
            <width>131</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;
font: 10pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Quên mật khẩu</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>110</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-left: 30px;
background-image: url(D:/NMCNPM/checking_attendance_system/fe/Image_and_icon/icons8-user-30.png); 
background-repeat: no-repeat;
background-position: left center; </string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>160</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-left: 30px;
background-image: url(D:/NMCNPM/checking_attendance_system/fe/Image_and_icon/icons8-password-30.png); 
background-repeat: no-repeat;
background-position: left center; </string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="eye">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>170</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-left: 30px;
background-image: url(D:/NMCNPM/checking_attendance_system/fe/Image_and_icon/icons8-eye-20.png); 
background-repeat: no-repeat;
background-position: left center; </string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="blind">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>170</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-left: 30px;
background-image: url(D:/NMCNPM/checking_attendance_system/fe/Image_and_icon/icons8-blind-20.png); 
background-repeat: no-repeat;
background-position: left center; </string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>75</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
